esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2375,6 +2381,9 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
         {
             RemoteFilename rfn;
             SocketEndpoint ep(destip);
+            if (ep.isNull())
+                throw MakeStringException(ECLWATCH_INVALID_INPUT, "Cannot resolve destination network IP from %s when despraying %s.", destip, srcname);
```

Comment:
minor: to be consistent with other new error messages, should this be "Despray to <dstname>: cannot ...." ?